foundation/oidc/src/main/kotlin/com/pingidentity/oidc/OidcClientConfig.kt

```diff
@@ -85,7 +85,7 @@ class OidcClientConfig {
     /**
      * Storage delegate for storing tokens.
```

Comment:
Now that this variable was changed from `StorageDelegated` to `Storage` we should rename the comment as well (?). . .

Reply:
Updated.